index.html

```diff
@@ -0,0 +1,23 @@
+<html>
+  <head>
+    <title>Photo Carousel</title>
+    <script>
+      class PhotoCarousel extends HTMLElement {
+        connectedCallback() {}
```

Comment:
so far we have only empty definition, without any realization/implementation on it. We will se it very soon.